src/apis/vaults.ts

```diff
@@ -367,6 +368,18 @@ export class DefaultVaultsAPI {
         }
     }
 
+    async getPremiumRedeemVaults(): Promise<Map<AccountId, PolkaBTC>> {
+        const customAPIRPC = this.api.rpc as any;
```

Comment:
We can use actual types now that we updated to the latest `polkadot{.js}`, so no `any`